easybuild/easyconfigs/c/csb/csb-1.2.5-foss-2020b.eb

```diff
@@ -0,0 +1,43 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/05
+easyblock = 'PythonPackage'
+
+name = 'csb'
+version = '1.2.5'
+
+homepage = 'https://github.com/csb-toolbox'
+description = """Computational Structural Biology Toolbox
+CSB is a python library and application framework, which can be used to solve   
+problems in the field of structural bioinformatics. If you are a                
+bioinformatician, software engineer or a researcher working in this field,      
+chances are you may find something useful here. Our package consists of a few   
+major components:
+1. Core class library - object-oriented, granular, with an emphasis on design   
+and clean interfaces.
+2. Application framework - console applications ("protocols"), which consume    
+objects from the core library in order to build something executable (and       
+hopefully useful).
+3. Test framework - ensures that the library actually works.
+"""
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/csb-toolbox/CSB/releases/download/R-%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3f05924acaca6673f94fc430965eabfe59a0c9b2cbdda8c1cb50a43ffe83f254']
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('matplotlib', '3.3.3'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/csb-%s' % x for x in [
+        'bfit', 'bfite', 'buildhmm', 'csfrag', 'embd',
+        'hhfrag', 'hhsearch', 'precision', 'promix', 'test']],
+    'dirs': ["lib/python%(pyshortver)s/site-packages"]
+}
+
```

Comment:
If one of the items in `bin` has a suitable option to use then can you add a `sanity_check_commands`?. So:
```
sanity_check_commands = ["csb-bfit --help"]
```

Reply:
done in fc2f252